src/musicbrainz/tagfetcher.cpp

```diff
@@ -165,7 +166,8 @@ void TagFetcher::slotAcoustIdTaskSucceeded(
 
         emit resultAvailable(
                 std::move(pTrack),
-                QList<mixxx::musicbrainz::TrackRelease>());
+                {},
+                tr("Could not identify track through Acoustid."));
```

Comment:
Nit: casing of `AcoustID`

Reply:
I have also fixed another occourenc, and will also fix Translations after merge.